.gitignore

```diff
@@ -40,4 +40,6 @@ kind_kubeconfig.yaml
 kind_kubeconfig_internal.yaml
 
 /operator-sdk
-/yq
\ No newline at end of file
+/yq
+
+# bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml
```

Comment:
Is this line required in .gitignore and if so, can we uncomment it?

Reply:
We will need it later when downstream will generate it.

Reply:
We can't ignore this file

Reply:
I'll add the new line but it wasn't there before.